tests/testnet.spec.js

```diff
@@ -694,3 +694,198 @@ describe('Record obt data, check', () => {
     expect(obtData.payee_fio_address).to.equal(testFioAddressName2)
   })
 })
+
+describe('Encrypting/Decrypting', () => {
+  const alicePrivateKey = '5J35xdLtcPvDASxpyWhNrR2MfjSZ1xjViH5cvv15VVjqyNhiPfa'
+  const alicePublicKey = 'FIO6NxZ7FLjjJuHGByJtNJQ1uN1P5X9JJnUmFW3q6Q7LE7YJD4GZs'
+  const bobPrivateKey = '5J37cXw5xRJgE869B5LxC3FQ8ZJECiYnsjuontcHz5cJsz5jhb7'
+  const bobPublicKey = 'FIO4zUFC29aq8uA4CnfNSyRZCnBPya2uQk42jwevc3UZ2jCRtepVZ'
+
+  const nonPartyPrivateKey = '5HujRtqceTPo4awwHAEdHRTWdMTgA6s39dJjwWcjhNdSjVWUqMk'
+  const nonPartyPublicKey = 'FIO5mh1UqE5v9TKdYm2Ro6JXCXpSxj1Sm4vKUeydaLd7Cu5aqiSSp'
+
+  let fioSDKBob = new FIOSDK(
+    bobPrivateKey,
+    bobPublicKey,
+    baseUrl,
+    fetchJson
+  )
+
+  it(`Encrypt/Decrypt - Request New Funds`, async () => {
+    const payeeTokenPublicAddress = bobPublicKey
+    const amount = 1.57
+    const chainCode = 'FIO'
+    const tokenCode = 'FIO'
+    const memo = 'testing encryption does it work?'
+    const hash = ''
+    const offlineUrl = ''
+
+    const content = {
+      payee_public_address: payeeTokenPublicAddress,
+      amount: amount,
+      chain_code: chainCode,
+      token_code: tokenCode,
+      memo,
+      hash,
+      offline_url: offlineUrl
+    }
+
+    const cipherContent = fioSDKBob.transactions.getCipherContent('new_funds_content', content, bobPrivateKey, alicePublicKey)
+   
+    expect(cipherContent).to.be.a('string')
+    // console.log(cipherContent)
+
+    // UKPHFo0xs3GwGAXMc44QqkDtj7dcbGHjU3cJmN1qiYWADvzyd9pen2WwKn0VZtk0ZTGFXpap7Id8nZxlCMK7TjkabO85XNbhausE4ZZzx3hm25bqV2GDRHpRomsRDGAzLbFEumsm+4UNBtnOqUK3Kuo91vKjlLIV3NoF83qOSbhL8QDqV2N/yJxSu4PsiDeqhhSypZx8McaubVoUueioWA==
+
+    const uncipherContent = fioSDKBob.transactions.getUnCipherContent('new_funds_content', cipherContent, alicePrivateKey, bobPublicKey)
```

Comment:
could you create `const newFundsContentStr = 'new_funds_content'` and use it everywhere?